test/unit/refresh.js

```diff
@@ -367,6 +272,93 @@ describe('swiftserver:refresh', function () {
     });
   });
 
+  describe('Generate scaffolded app from an valid swagger URL', function () {
+
+    var runContext;
+    var error;
+    var server;
+
+    before(function () {
+        var http = require('http');
+        var swagger = fs.readFileSync(path.join( __dirname, '../resources/person_dino.json'), 'utf8');
+        server = http.createServer(function(request, response) {
+          response.writeHead(200, {'Content-Type': 'application/json' });
+          response.end(swagger);
+        }).listen(8080);
+
+        // Mock the options, set up an output folder and run the generator
+        var spec = {
+          appType: 'scaffold',
+          appName: appName,
+          fromSwagger: 'http://localhost:8080/stuff',
+          config: {
+            logger: 'helium',
+            port: 4567
+          }
+        };
+      runContext = helpers.run(path.join( __dirname, '../../refresh'))
+        .withOptions({
+          specObj: spec
+        })
+      return runContext.toPromise().catch(function(err) {
```

Comment:
I don't think we want a catch here, it will swallow errors because the `error` value is never tested. We don't really need this at all, since we don't want to ensure an error is thrown as part of any of the tests in this describe block.